bcp47-orphans/tests/Data/BCP47/AesonSpec.hs

```diff
@@ -0,0 +1,13 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Data.BCP47.AesonSpec (spec) where
```

Comment:
:art: 

```hs
module X
  ( y
  )
where
```

Our current Guides only make an exception for `Main (main)`, though I wouldn't veto adding an exception for `Spec (spec)` too. It seems it's ingrained in many folks' muscle memory.

Reply:
I think I probably copy/pasted this from the test module in `bcp47`, but I went ahead and updated the ones in `-orphans`